lib/awful/spawn.lua

```diff
@@ -370,6 +370,18 @@ function spawn.easy_async(cmd, callback)
     })
 end
 
+--- Call spawn.easy_async with a shell.
+-- This calls `cmd` with `$SHELL -c` (via `awful.util.shell`).
+-- @tparam string|table cmd The command.
+-- @function callback The callback function.
```

Comment:
You forgot to copy/paste the arguments here. Sorry about being picky, it's just that correct/consistent doc is something we try to take seriously now.